.travis.yml

```diff
@@ -9,11 +10,12 @@ before_install:
   - go get -t -v ./...
 
 script:
+  - go vet ./...
   - go test -race -coverprofile=coverage.txt -covermode=atomic
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
-  
```

Comment:
This was spurious whitespace that my editor deleted automatically. I left it in this PR since its a small fix :-)